### PR TITLE
Add Test Pager Duty Secret Verification

### PR DESCRIPTION
**Pull request by milindw96**
Signed-off-by: milindw96 mwaykole@redhat.com

### CODE REVIEW

File: tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duity.robot [hunk 1]
```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Documentation   RHODS_Pager_Duity_key__VERIFICATION
```
Comment (jgarciao) [inline tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duity.robot:2]: There is a typo in the filename and other places of this file (it's "pager duty", not "duity")

File: tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duity.robot [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+*** Test Cases ***
+PagerDuty Dummy Secret Verification
+     [Documentation]    Verification of PagerDuty Secret
+     [Tags]  Sanity
+     ...     ODS-737
+     ${service_key}   Get Service Key From Alertmanager ConfigMap
+     ${secret_key}    Get Secert Key From Secrets
+     Should Be Equal As Strings    ${service_key}   ${secret_key}
```
Comment (jgarciao) [inline tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duity.robot:31]: In Polarion  title is ODS-737 - Verify PagerDuty dummy secret is correctly stored in when installing RHODS **not** using the add-on flow

That means that:
- You need to add the tag **Deployment-Cli**  (I just added for you in the style guide)
- The secret will be always the same, so you can check the value of it to verify it's a dummy one. Maybe you can add this step to Polarion

File: tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duity.robot [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Get Service Key From Alertmanager ConfigMap
+     [Documentation]    Get Service Key From Alertmanager ConfigMap
+
+     ${c_data}   OpenShiftCLI.Get  kind=ConfigMap  namespace=${NAMESPACE}   field_selector=metadata.name==${CONFIGMAP_NAME}
+     ${a_data}    Set Variable     ${c_data[0]['data']['alertmanager.yml']}
+     ${match_list}      Get Regexp Matches   ${a_data}     ${REGEX_PATTERN}
+     ${key}       Split String    ${match_list[0]}
+     [Return]     ${key[-1]}
+
+Get Secert Key From Secrets
```
Comment (jgarciao) [inline tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duity.robot:44]: Maybe rename it to `Get PagerDuty Key From Secrets`

File: tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duty.robot [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+     ...     Deployment-Cli
+     ${service_key}   Get PagerDuty Key From Alertmanager ConfigMap
+     ${secret_key}    Get PagerDuty Key From Secrets
+     Should Be Equal As Strings    ${service_key}   ${secret_key}
+
+
+*** Keywords ***
+Get PagerDuty Key From Alertmanager ConfigMap
+     [Documentation]    Get Service Key From Alertmanager ConfigMap
+
```
Comment (lugi0) [inline tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duty.robot:37]: remove blank space

File: tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duty.robot [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+*** Test Cases ***
+PagerDuty Dummy Secret Verification
+     [Documentation]    Verification of PagerDuty Secret
+     [Tags]  Sanity
+     ...     ODS-737
+     ...     Deployment-Cli
+     ${service_key}   Get PagerDuty Key From Alertmanager ConfigMap
+     ${secret_key}    Get PagerDuty Key From Secrets
+     Should Be Equal As Strings    ${service_key}   ${secret_key}
```
Comment (lugi0) [inline tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duty.robot:31]: Add check on dummy value for either service_key or secret_key

**Review by lugi0 — APPROVED** [review]

**Review by bdattoma — APPROVED** [review]
lgtm

**Pull request merged**